pkg/cloudprovider/providers/azure/azure_test.go

```diff
@@ -821,6 +821,13 @@ func TestSplitProviderID(t *testing.T) {
 	}
 }
 
+func TestMetadataURLGeneration(t *testing.T) {
+	fullPath := MakeMetadataURL("some/path")
+	if fullPath != "http://169.254.169.254/metadata/some/path" {
```

Comment:
Can we use constant metaURL here to check the full path so that it will not break the test if the const is changed?